Docs/source/running_cpp/parameters.rst

```diff
@@ -503,12 +503,12 @@ Particle initialization
     Whether or not to use OpenMP threading for particle initialization.
 
 * ``<species>.do_field_ionization`` (`0` or `1`) optional (default `0`)
-    Do field ionization for this species (using the ADK theory). Currently,
-    this is slow on GPU.
+    Do field ionization for this species (using the ADK theory).
 
 * ``<species>.physical_element`` (`string`)
     Only read if `do_field_ionization = 1`. Symbol of chemical element for
     this species. Example: for Helium, use ``physical_element = He``.
+    Elements up to atomic number Z=86 (Radon) are supported, let us know if you need higher Z.
```

Comment:
Maybe the suggestion can be even more specific
```suggestion
    Elements up to atomic number Z=86 (Radon) are supported. Please open an issue on GitHub, if you need higher Z so we can implement them.
```

Reply:
Isn't the highest Z=18 and we add some noble gases on top?
https://github.com/ECP-WarpX/WarpX/blob/master/Source/Utils/IonizationEnergiesTable.H#L41-L44